ShortcutTrainerBackend/ShortcutTrainerBackend/Data/Models/Question.cs

```diff
@@ -7,4 +8,13 @@ public class Question
     public required int Id { get; set; }
     public required string Content { get; set; }
     public required string Shortcut { get; set; } // ToDo: Format like "strg+c"
+    public required QuestionStatus Result { get; set; }
+    public required List<QuestionParameter> QuestionParameter { get; set; }
```

Comment:
QuestionParameter ist keine Eigenschaft von Question als Modell-Klasse. QuestionParameter ist die Mapper-Klasse für  die Routen im QuestionsController.
Ein Property in der Klasse QuestionParameter hat auch einen Typo